packages/config/src/config.js

```diff
@@ -13,22 +54,26 @@ module.exports = ({
     useFileHash = true,
     betaEnv = 'ci',
     sassPrefix,
-    deployment
+    isProd,
+    isStandalone,
+    standalonePort,
+    proxyConfig
 } = {}) => {
     const filenameMask = `js/[name]${useFileHash ? '.[chunkhash]' : ''}.js`;
     return {
-        mode: mode || (process.env.NODE_ENV === 'production' ? 'production' : 'development'),
-        devtool: 'source-map',
+        mode: mode || (isProd ? 'production' : 'development'),
+        devtool: isProd ? 'source-map' : 'cheap-module-source-map',
```

Comment:
`source-map` is expensive for dev. I've opted for `cheap-module-source-map` instead.